### PR TITLE
DHFPROD-4589: Use custom trigger for collapsable side bar

### PR DESCRIPTION
**Pull request by timur-isangulov**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx [hunk 1]
```diff
@@ -76,6 +76,14 @@ describe('json scenario for snippet on browse documents page', () => {
...
 
   it('apply facet search and verify docs, hub/entity properties', () => {
+    //verify collapsable slider functionality
+    browsePage.getSideBarCollapseIcon().should('be.visible');
+    browsePage.getSideBarContainer().should('be.visible');
+    browsePage.getSideBarCollapseIcon().click();
+    browsePage.getSideBarContainer().should('not.be.visible');
+    browsePage.getSideBarCollapseIcon().click();
+    browsePage.getSideBarContainer().should('be.visible');
+
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx:86]: I have pushed an RTL test to verify collapsible sidebar, so you can get rid of this test in e2e.
Please **fetch my changes first**, so your repo gets updated.
git fetch timur-isangulov
git rebase timur-isangulov/DHFPROD-4589

**Comment by bsrikan** [comment]
@xnikhil08 FYI. I have added an RTL test for browse page. Lets see if we can build up on this and have others follow suit.

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx (hunk 1 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx]: Removed. Thanks.

File: marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx [hunk 2]
```diff
@@ -95,6 +95,11 @@ class BrowsePage {
    * available facets are 'collection', 'created-on', 'job-id', 'flow', 'step'
    */
 
+  
+  getSideBarContainer() {
+    return cy.get('#sideBarContainer');
```
Comment (Sanjeevani19) [inline marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx:100]: @timur-isangulov Please remove these methods too which are not being used

Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx]: Removed.

File: marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx [hunk 3]
```diff
@@ -214,6 +219,10 @@ class BrowsePage {
     return cy.get('[data-cy=table-view]').click();
   }
 
+  getSideBarCollapseIcon() {
+    return cy.get('#sidebar-collapse-icon');
```
Comment (Sanjeevani19) [inline marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx:218]: @timur-isangulov is this method being used ?

Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx:218]: This function might be useful for the future tests. Srikanth made a commit with the test, probably shouldn't squash.

**Pull request merged**